<launch>  
    <include file="$(find interbotix_xsarm_control)/launch/xsarm_control.launch" >
        <arg name="robot_model" value="px100" />
        <arg name="use_sim" value="false" />
    </include>

    <node name="brl_pxh_rack" pkg="brl_pxh_api" type="brl_pxh_rack.py" output="screen" />
</launch>
